iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/AttachLoadOnVoltageLevel.java

```diff
@@ -0,0 +1,171 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.modification.topology;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.modification.NetworkModification;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.extensions.BusbarSectionPosition;
+
+import java.util.List;
+
+import static com.powsybl.iidm.modification.topology.TopologyModificationUtils.*;
+
+/**
+ * This method adds a new load on an existing voltage level. The voltage level should be described
+ * in node/breaker.
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public class AttachLoadOnVoltageLevel implements NetworkModification {
+
+    //Load attributes
+    private final String loadId;
+    private final double p0;
+    private final double q0;
+    private final LoadType loadType; //add constant
+
+    //VoltageLevel attributes
+    private final String voltageLevelId;
+    private String bbsId; //Id of the busBar where the switch will be open
+
+    private List<String> bbsIds;
+
+    public AttachLoadOnVoltageLevel(String loadId, double p0, double q0,
```

Comment:
This is not the right design because:
 - the idea is not to duplicate the IIDM adder which already do most of the job
 - but just kind of extend adder capabilities to create a full feeder cell instead of just the injection
 - we don't want to duplicate all this for each of the network equipment, it should be generic to injections and branches.

It has to look like:
```java
Network n = ...
LoadAdder adder = vl.newLoadAdder
  .setId(...)
 // WE DON'T NEED TO SET THE NODE !!!!!!!
  ...
.add()
new InjectionFeederCallCreation(adder, "BBS", CellType::ONE_BREAKER) // busbar section could be optional, and if not defined automatically choosen
   .apply(n);
```

Reply:
Also one very important thinks when it comes to network modification is functional logging, otherwise there is no way to known what has been done especially in loose mode when it case of equipment not done where there is no exception.
So we need:
  - to create a report when modification is ok to explain what has been done
  - to create a report when modification has not been done to explain exactly what was wrong

@annetill also we need to to the same in AttachNewLineOnLine and AttachVoltageLevelOnLine. In gridsuite integration we added functional logs externally but this is not the right way to do, it should have been done inside powsybl. Also in case of error this is the only place to describe what was wrong.